worlds/apsudoku/docs/setup_en.md

```diff
@@ -13,25 +11,33 @@ Does not need to be added at the start of a seed, as it does not create any slot
 
 ## Installation Procedures
 
-Go to the latest release from the [APSudoku Releases page](https://github.com/EmilyV99/APSudoku/releases). Download and extract the `APSudoku.zip` file.
+Go to the latest release from the [APSudoku Releases page](https://github.com/APSudoku/APSudoku/releases). Download and extract the appropriate file for your platform.
 
 ## Joining a MultiWorld Game
 
-1. Run APSudoku.exe
-2. Under the 'Archipelago' tab at the top-right:
-	- Enter the server url & port number
+1. Run the APSudoku .exe (name varies by platform)
+2. Under `Settings`->`Connection` at the top-right:
+	- Enter the server ip & port number
 	- Enter the name of the slot you wish to connect to
 	- Enter the room password (optional)
 	- Select DeathLink related settings (optional)
-	- Press connect
-3. Go back to the 'Sudoku' tab
-	- Click the various '?' buttons for information on how to play / control
-4. Choose puzzle difficulty
-5. Try to solve the Sudoku. Click 'Check' when done.
+	- Press `Connect`
+3. Optional things:
+	- You can check other settings under `Settings`->`Sudoku`, and can change the colors used under `Settings`->`Theme`.
```

Comment:
Having the line able to break up reduces accessibility, this is better (and the other two arrows)
```suggestion
	- You can check other settings under `Settings` &rarr; `Sudoku`, and can change the colors used under `Settings`->`Theme`.
```